src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,65 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                id:
+                    description: Annotation ID
+                    type: string
+                access:
+                    $ref: ../misc/access-level.v1.yaml
+                hasParent:
```

Comment:
is there a way to find out the `id` or something similar for the parent annotations? Rather than only knowing if it has one? This to promote reusability of this API with a low effort.

Reply:
All the other field names seem to match the other schemas in this repository

Reply:
In the annotation schema the `references` property lists the annotation id of all parents.

https://github.com/hypothesis/h/blob/75714a4829bf17cdb760b98829942b2646e60542/docs/_extra/api-reference/schemas/annotation-schema.json#L25-L30

I will introduce this array. The last entry in the array is the immediate parent and the first entry is the top level annotation or page note. If it is empty, it has no parents and is a top-level annotation or page note.